### PR TITLE
Add information content file as a dependency during SEMSIM profile generation. Fixes #27.

### LINKED ISSUE #27: Add information content file as a dependency during SEMSIM profile generation
When calculating SEMSIM profiles using --information content, the pipeline must ensure the information content file is generated beforehand to avoid errors during OAK command execution. Declare the information content file as a dependency in the semsim profile Makefile goal.

### PR DESCRIPTION
**Pull request by souzadevinicius**
Fixes #27

### CODE REVIEW

File: Makefile.j2 [hunk 1]
```diff
@@ -65,7 +65,7 @@ run-semsim: {{ directories.profiles }}/{{ semsim.ontology }}-{{ semsim.subset }}
 {% if 'ic' in semsim -%}
 
 
-{{ directories.profiles }}/{{ semsim.ontology }}-{{ semsim.subset }}.{{semsim.threshold}}.{{ semsim.method }}.ic.tsv: $(ONTOLOGYDIR)/{{ semsim.ontology }}.db $(TMP_DATA)/{{ semsim.prefixes.subject }}_terms.txt $(TMP_DATA)/{{ semsim.prefixes.object }}_terms.txt
+{{ directories.profiles }}/{{ semsim.ontology }}-{{ semsim.subset }}.{{semsim.threshold}}.{{ semsim.method }}.ic.tsv: $(ONTOLOGYDIR)/{{ semsim.ontology }}.db $(TMP_DATA)/{{ semsim.prefixes.subject }}_terms.txt $(TMP_DATA)/{{ semsim.prefixes.object }}_terms.txt $(TMP_DATA)/{{ semsim.ic }}
```
Comment (matentzn) [inline Makefile.j2:68]: Missing extension, e.g. `.tsv`?

Comment (souzadevinicius) [reply Makefile.j2:68]: No. This value comes specifically from the 'ic:' property in `profiles.yml` file, where we set the file path name along with the extension.

Comment (matentzn) [reply Makefile.j2:68]: Ok!

**Review by matentzn — APPROVED** [review]
lgtm

**Pull request merged**